atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -509,7 +509,8 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
+                    'autorebuild': is_rebuild(self.workflow)
```

Comment:
A build will have multiple archives when multi-arch is in the picture. In this case, each archive is bundled together in a single build. The case of having a mix of archives in a single build that were created via auto rebuild and manually, is not valid.